src/PhpSpreadsheet/Writer/Xlsx/Chart.php

```diff
@@ -1144,6 +1144,15 @@ private function writePlotGroup($plotGroup, $groupType, $objWriter, &$catIsMulti
                 if ($groupType == DataSeries::TYPE_STOCKCHART) {
                     $objWriter->startElement('a:noFill');
                     $objWriter->endElement();
+                } elseif ($plotLabel) {
+                    $fillColor = $plotLabel->getFillColor();
+                    if ($fillColor !== null && !is_array($fillColor)) {
```

Comment:
I'd go for the more straightforward:

```suggestion
                    if (is_string($fillColor)) {
```

Reply:
Totally agree. Commited, thanks